airbyte-integrations/connectors/source-google-analytics-v4/source_google_analytics_v4/source.py

```diff
@@ -403,7 +404,7 @@ def parse_response(self, response: requests.Response, **kwargs: Any) -> Iterable
                         record[metric_name.replace("ga:", "ga_")] = value
 
                 record["view_id"] = self.view_id
-
+                record["isDataGolden"] = report.get("data", {}).get("isDataGolden", True)
```

Comment:
this should be false by default, from the [google docs](https://developers.google.com/analytics/devguides/reporting/core/v4/basics): 

> Note: When parsing the response body, if a boolean field is absent, you can assume its value is false. Only boolean fields with true value are shown in the response; for example, the [isDataGolden](https://developers.google.com/analytics/devguides/reporting/core/v4/rest/v4/reports/batchGet#ReportData.FIELDS.is_data_golden) field will be in the response if its value is true.

Reply:
We should also add a unit test which mocks the API response and verifies that if the API doesn't return isdatagolden then its value should be false in the outpu

Reply:
Thanks, that's a good point. I was confused by this [line](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-google-analytics-v4/source_google_analytics_v4/source.py#L410), now both fixed. Unit test also added